roles/sp/tasks/tng-policy-mngr.yml

```diff
@@ -0,0 +1,20 @@
+---
+# Running the 5GTango POLICY MANAGER (Docker container)
+- debug: msg="##### SP POLICY MANAGER DEPLOYMENT {{ lookup('pipe', 'date +%Y%m%d-%H%M') }}"
+- name: POLICY MANAGER - running Docker containers
+  docker_container:
+    name: tng-policy-mngr
+    image: "{{ repository }}/tng-policy-mngr:{{ version }}"
+    state: started
+    recreate: yes
+    pull: true
+    restart_policy: "always"
+    network_mode: bridge
+    networks:
+      - name: "{{ docker_network_name }}"
+        aliases:
+          - tng-policy-mngr
+        links:
```

Comment:
@efotopoulou you don't need the link since it is on the same docker network. Can you remove it from the playbook?